src/components/ui/Icon/Icon.tsx

```diff
@@ -5,7 +5,7 @@ import sprites from '@assets/sprites.svg';
 import style from './Icon.module.scss';
 
 interface IconProps {
-  iconName: string;
+  iconName: 'plus' | 'delete' | 'close';
```

Comment:
Try to use enums

Reply:
Decomposed iconName to IconsTypes enum. Updated relevant components.